docs/src/docs/asciidoc/user/resilience.adoc

```diff
@@ -0,0 +1,189 @@
+---
+---
+= Resilience
+
+*Note:* This is not the current state of things. It is the current ongoing effort to improve cluster resilience.
+
+== What can go wrong
+
+* Server fails to get an answer erratically because of network or something
+* Server fails to send messages to the client erratically
+* Server goes down
+* Server failover
+* Loader-writer backend fails
+
+== Strong vs Eventual
+
+Strong means: If the client ask for a value that was updated by some other server, it will always get the
+latest value.
+
+In particular, it means that as soon as an update from the server might be missed, we need to clear
+the caching tier. It also means that if we fail to set a value to the authoritative tier, we can't use
+the cache anymore. On this key at least.
+
+Eventual means the correct value will eventually be there. So if we loose contact with the server, it is possible to keep
```

Comment:
lose not loose :-)

Reply:
Done :-) And a bunch of other grammar errors as well